README.md

```diff
@@ -4,27 +4,43 @@ Automating MakePlayingCards's online ordering system.
 * Elasticsearch (probably easiest with Docker)
 * A Google account
 
 # Setup
-1. Clone this repo somewhere on your server
-2. In the same directory as the repo, create a folder called `staticroot` for static assets
-3. Deploy the Django project (I'm using DigitalOcean for Ubuntu) with a webserver (I'm using Apache) and serve static files with another webserver if you want (I was previously using nginx but now I just serve static files with Apache as well)
-4. Run Elasticsearch
-5. Set up your Google Drive credentials in the `MPCAutofill` directory (base Django directory). You should set up a Google Drive service account and store your credentials as `client_secrets.json`
-6. Run the command `manage.py import_sources` to sync sources in `drives.csv` to database, and `manage.py update_database` to populate the database (optionally specifying a particular drive to sync with `-d <drivename>`)
-7. Create a cronjob to periodically run the database updater command, to ensure MPC Autofill reflects the current state of the linked Drives, and another cronjob to periodically synchronise the double-faced cards table with Scryfall:
-* `0 0 * * * bash /root/mpc-autofill/update_database >> /root/db_update.txt 2>&1`
-* `0 0 * * SUN bash /root/mpc-autofill/sync_dfcs`
-8. Deploy two Google Script according to the code specified in `autofill.py` and adjust the URLs in that script to point to your GS endpoints
+#### <em>All steps with the prefix [PROD] are only needed if you are deploying this web app in a production environment. For running the application locally or for development, you can skip these steps.</em>
+1. Clone this repo somewhere on your server/local computer. This will be referred to as the `$working_directory`.
+2. If you are using an IDE to run the Django commands and are NOT using Windows: go to the `$working_directory/.run` directory and for each XML file delete the option `SDK_HOME`
+3. Add the file `$working_directory/MPCAutofill/MPCAutofill/.env` to edit any environment variables used in `$working_directory/MPCAutofill/MPCAutofill/settings.py`.  
+   IN ORDER TO RUN THE APP LOCALLY, YOU MUST ADD `DJANGO_DEBUG=on` TO THE `$working_directory/MPCAutofill/MPCAutofill/.env` FILE
+4. **[PROD]** In the `$working_directory`, create a folder called `staticroot` for static assets
+5. **[PROD]** Deploy the Django project (I'm using DigitalOcean for Ubuntu) with a webserver (I'm using Apache) and serve static files with another webserver if you want (I was previously using nginx but now I just serve static files with Apache as well)
+6. Install Elasticsearch based on your OS, or with Docker ([Installation Guide](https://www.elastic.co/guide/en/elasticsearch/reference/current/install-elasticsearch.html))
+   1. It might make the most sense to unzip the Elasticsearch directory into your `$working_directory`
+   2. Optionally, add the executables to your PATH for easy terminal access (add `location-of-package/elasticsearch-<VERSION_NUM>/bin` to your PATH)
```

Comment:
This made no sense to someone on Windows. Maybe include this?
```suggestion
   2. Optionally, add the executables to your PATH for easy terminal access (add `location-of-package/elasticsearch-<VERSION_NUM>/bin` to your PATH)
   3. For Windows, create a shortcut in your `$working_directory' for easy terminal access by opening `unzipped_elasticsearch_folder/bin` and create a shortcut to the `elasticsearch.bat` file
```

Reply:
I guess I didn't figure one would run the Elasticsearch executable via GUI. I think the optimization of a shortcut to the `bin` directory might be more roundabout than just an alias of the EXE file.